src/phantom.js

```diff
@@ -75,9 +76,15 @@ export default class Phantom {
         this.process.on('error', error => {
             logger.error(`Could not spawn [${phantomjs.path}] executable. Please make sure phantomjs is installed correctly.`);
             logger.error(error);
+            this.kill('Process got an error');
             process.exit(1);
         });
 
+        this.process.stdin.on('error', (e) => {
+            logger.debug(`Child process received error ${e}, sending kill signal`);
+            this.kill('Error reading from stdin');
```

Comment:
I think you are right. `stderr` will get used instead which I handled correctly. 👍